docs/src/content/docs/guides/connectionoptions.mdx

```diff
@@ -85,37 +129,92 @@ It only takes two steps to make an outgoing call to regular phone numbers throug
                         <Stream url="${joinUrl}"/>
                     </Connect>
                 </Response>`,
-        to: phoneNumber, // the number you are calling
-        from: twilioPhoneNumber // your twilio number
+        to: phoneNumber,
+        from: twilioPhoneNumber
     });
     ```
 </Steps>
 
-See the [twilio-outgoing-call](https://github.com/fixie-ai/ultradox/tree/main/examples/twilio-outgoing-call) example for more.
-
-This example shows one of the many options Twilio provides for making outgoing calls. Consult the [Twilio docs](https://www.twilio.com/docs) for more details.
-
-### Incoming Calls
-Incoming calls require essentially the same two steps as outgoing calls:
-
+#### Incoming Calls with Twilio
 <Steps>
-1. **Create an Ultravox Call** → Create a new call (see [above](#creating-a-phone-call-with-twilio)), and get a `joinUrl`. *Note: for incoming calls you will want to keep `firstSpeaker` set to the default ("FIRST_SPEAKER_AGENT").*
+1. **Create an Ultravox Call** → Create a new call with `medium: { "twilio": {} }` and `firstSpeaker` set to "FIRST_SPEAKER_AGENT".
 
-1. **Receive Inbound Twilio Phone call** → Use the `joinUrl` with a Twilio [`<Stream>`](https://www.twilio.com/docs/voice/twiml/stream).
+2. **Handle Inbound Call** → Use the `joinUrl` with a Twilio `<Stream>`:
 
     ```xml
-    <!-- Example TwiML Response -->
     <?xml version="1.0" encoding="UTF-8"?>
     <Response>
         <Connect>
             <Stream url="your_ultravox_join_url" />
         </Connect>
     </Response>
+    ```
+</Steps>
+
+For more details, see the [Twilio documentation](https://www.twilio.com/docs).
+</TabItem>
+
+<TabItem label="Plivo">
+
+#### Outgoing Calls with Plivo
+<Steps>
+1. **Create an Ultravox Call** → Create a new call as shown above with `medium: { "plivo": {} }`, `firstSpeaker: "FIRST_SPEAKER_USER"`, and get a `joinUrl`.
+
+2. **Initiate Plivo Phone call** → Use the `joinUrl` with AudioStream:
+
+    ```javascript
+    // Example using the plivo node library
+    const call = await plivo.calls.create({
+      to: phoneNumber,
+      from: plivoPhoneNumber,
+      answer_url: answerUrl, // URL that returns the XML below
+      answer_method: "GET"
+    });
+    ```
+
+    The answer URL should return:
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Response>
+        <Stream url="${joinUrl}" 
+                audioTrack="both" 
+                contentType="audio/x-l16;rate=8000" 
+                bidirectional="true" />
```

Comment:
```suggestion
        <Stream keepCallAlive="true" 
                contentType="audio/x-l16;rate=16000"
                bidirectional="true">
            ${joinUrl}
        </Stream>
```

Maybe also worth noting that any contentType supported by Plivo will be fine for Ultravox, but we recommend audio/x-l16;rate=16000 for the best audio quality.